src/app/prescription/medication-request-form/dosage-instruction-form/dosage-instruction-form.component.css

```diff
@@ -7,13 +11,15 @@
 }
 
 button.mat-stroked-button {
-  height: 51px;
+  height: 32px;
```

Comment:
Here it's ok as its just a demo tool but usually try to not use px as its not very responsive